src/main/scala/io/iohk/ethereum/blockchain/sync/SyncController.scala

```diff
@@ -68,8 +69,9 @@ class SyncController(
   }
 
   def startFastSync(): Unit = {
-    val fastSync = context.actorOf(FastSync.props(fastSyncStateStorage, appStateStorage, blockchain, validators,
-      peerEventBus, etcPeerManager, syncConfig, scheduler), "fast-sync")
+    val fastSync = context.actorOf(FastSync.props(fastSyncStateStorage, appStateStorage,
+      new FastSyncStateHandler(blockchain, validators, syncConfig, appStateStorage, Random.nextInt), peerEventBus, etcPeerManager, syncConfig, scheduler),
```

Comment:
maybe extract `new FastSyncStateHandler(blockchain, validators, syncConfig, appStateStorage, Random.nextInt)` to value and probably all params would fit one line and it would be more readable